app/views/users/_following.html.erb

```diff
@@ -0,0 +1,34 @@
+<ul class="accordion" data-accordion data-allow-all-closed="true">
+
+  <% @followable_types.each do |followable_type| %>
+
+    <li class="accordion-item" data-accordion-item>
+
+      <a href="#" class="accordion-title">
+        <span class="icon">
+          <i class="icon icon-<%= entity_icon(followable_type) %>"></i>
+        </span>
+        <span class="title">
+          <strong><%= entity_title(followable_type) %></strong>
+        </span>
+      </a>
+
+      <div class="accordion-content" data-tab-content>
+        <table>
+          <tbody>
+            <% @follows.where(followable_type: followable_type).each do |follow| %>
+              <% if followable_type == "Proposal" %>
```

Comment:
In the scenario were another entity becomes followable, for both icon and title translation following the trail to the helper will be needed, so I don't see it as a big downside to the proposed refactor. I just imagine 4-5 entities being followable and the if-elsif-elsif.. being too big when just following a convention (that also complains with how rails expects things to be named) could make it easier.

But thats like you say a subjective matter, I prefer conventions over more lines, maybe @voodoorai2000 can take a look and say if he has a hard opinion about it or not.